Dockerfile-linux.template

```
@@ -50,7 +50,6 @@ RUN apk add --no-cache \
 		curl \
 		tar \
 		xz \
-# https://github.com/docker-library/php/issues/494
 		openssl
```

Without the comment, we should re-alphabetize `openssl` into the list above:

```suggestion
		curl \
		openssl \
		tar \
		xz
```